controllers/apps/components/internal/component_base_stateful.go

```diff
@@ -419,6 +423,25 @@ func (c *StatefulComponentBase) updatePVCSize(reqCtx intctrlutil.RequestCtx, cli
 	return nil
 }
 
+// statusUpdateTemplateRenderedConfig transform the action of rendered config by config-template from noop to update in cluster Status reconciliation
+func (c *StatefulComponentBase) statusUpdateTemplateRenderedConfig() error {
+	for _, v := range ictrltypes.FindAll[*corev1.ConfigMap](c.Dag) {
+		node, _ := v.(*ictrltypes.LifecycleVertex)
+		cm, _ := node.Obj.(*corev1.ConfigMap)
+		_, ok := cm.GetLabels()[constant.CMConfigurationTypeLabelKey]
```

Comment:
When will this annotation be deleted, or the CM object will be updated/created on every reconciliation?

Reply:
in every reconciliation, the CM object isn't depended on label/annotation。 it is controlled by config-manager.
this label or annotation just indicates some meta data